src/Containers.jsx

```diff
@@ -129,6 +137,10 @@ const ContainerActions = ({ con, container, onAddNotification, localImages, upda
                 });
     };
 
+    const restartQuadletContainer = () => {
+        utils.systemctl_spawn(["restart", container.Config.Labels.PODMAN_SYSTEMD_UNIT], container.uid === 0);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2286-9d63ad0b-20251017-133332-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)